app/controllers/AmendCaseSendInformationController.scala

```diff
@@ -88,7 +88,7 @@ class AmendCaseSendInformationController @Inject()(
     renderFileVerificationStatus(reference, request.userAnswers.fileUploadState)
   }
 
-  final def removeFileUploadByReference(reference: String): Action[AnyContent] = (identify andThen getData andThen requireData).async { implicit request =>
+  final def removeFileUploadByReference(reference: String, mode : Mode = NormalMode): Action[AnyContent] = (identify andThen getData andThen requireData).async { implicit request =>
```

Comment:
@somaraja we've added this parameter but it's not used. Do we need it?

Reply:
@lmugnai we need this for CYA page.